docs/backtesting.md

```diff
@@ -72,6 +72,17 @@ The exported trades can be used for [further analysis](#further-backtest-result-
 freqtrade backtesting --export trades --export-filename=backtest_samplestrategy.json
 ```
 
+#### Supplying custom fee value
+
+Sometimes your account has certain fee rabates, which are not visible to ccxt.
```

Comment:
rebates?

Reply:
take binance as example...

once you reach a certain volume per month, you pay fewer fees (0.0020 instead of 0.0025). freqtrade doesn't account for this at the moment.

I tried to explain this in the docs now ...